sbc_compassion/data/lang_data.xml

```diff
@@ -0,0 +1,73 @@
+<openerp>
+    <data>
+        <record id="0" model="res.lang.compassion">
```

Comment:
give better ids than number to xml records. Keep in mind that xml_ids are not the resulting ids in the database, they are just "internal names" for referencing the created records in the database. Something like `lang_compassion_english` would be more appropriate.